node-feature-discovery-job.json.template

```diff
@@ -17,6 +17,14 @@
         }
       },
       "spec": {
+        "volumes": [
```

Comment:
@thed00de responded via email but it didn't get linked to this thread (see comments below)